### ISSUE TITLE
New Kali install kismet apt update

### ISSUE THREAD

**Issue by tmb2077** [greatscottgadgets/ubertooth#512]
### Steps to reproduce
1.kismet can't find plugun
2. Sudo apt update won't work
3.chang directory

### Expected behaviour
Run kismet for first time, and do an apt udate

### Actual behaviour
I just installed Kali on external hard drive, suspect paths are messed up, I run kismet it tries to pull from wrong file place and there is no kismet plugin

### Version information
Kali newest version

**Ubertooth tools version (ubertooth-rx -V):**

**libbtbb version:**

**Ubertooth firmware version (ubertooth-util -v):**

If you are reporting a problem that involves third party software 
(Wireshark/Kismet/etc), please report the version here.

### Output
```
Insert any commandline or build output here
```

**Comment by straithe** [comment]
Have you opened an issue with Kismet?

**Comment by tmb2077** [comment]
Yes I figured it out, I was running Kali Persistence and I needed to do a full install on my external hard drive to get all the extra ins and outs.   Sent from Mail for Windows From: StraitheSent: Monday, January 2, 2023 9:26 PMTo: greatscottgadgets/ubertoothCc: tmb2077; AuthorSubject: Re: [greatscottgadgets/ubertooth] New Kali install kismet apt update (Issue #512) Have you opened an issue with Kismet?—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you authored the thread.Message ID: ***@***.***>

**Comment by straithe** [comment]
Thank you for the update. I will close this issue as resolved.
